package.json

```diff
@@ -42,7 +42,9 @@
     "react-leaflet": "^3.2.0",
     "react-multi-carousel": "^2.6.3",
     "react-share": "^4.4.0",
+    "reactour": "^1.18.6",
     "simplebar-react": "^2.1.0",
+    "styled-components": "^5.3.1",
```

Comment:
Why?

Reply:
the component has a peer dependency on it